lib/cmd/fh2/account.js

```diff
@@ -140,24 +141,30 @@ function uploadResourcesBatch(destination, filepath, cb) {
           }
         }
         var filesToUpload = [];
-        for (var dest in fileTypes) {
-          for (var config in fileTypes[dest]) {
-            for (var filetype in fileTypes[dest][config]) {
-              if (fileTypes[dest][config][filetype].length > 1) {
-                log.warn("Found more than 1 resources for destination " + dest + " , config : " + config + " , type : " + filetype + ". Ignored.");
-              } else {
-                filesToUpload.push((function (path, dest, config, filetype) {
-                  return function (callback) {
-                    uploadResource(dest, path, filetype, config, function (err, data) {
-                      callback(err, data);
-                    });
-                  };
-                })(fileTypes[dest][config][filetype][0], dest, config, filetype));
+        for (var destination in fileTypes) {
+          if (fileTypes.hasOwnProperty(destination)) {
```

Comment:
I was kinda afraid of doing that for this big nesting and introducing a bug, but I can give it a try.
Do you know if this is covered by the existing unit tests?